_posts/0000-01-02-Felorina.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+It's one big party up in here!
```

Comment:
Add an image here for maximum flavor.
![db9u8o9](https://user-images.githubusercontent.com/45719432/89720592-a508c980-d9a1-11ea-9d3a-7bb468dcef20.png)